From: uazo <uazo@users.noreply.github.com>
Date: Mon, 13 Nov 2023 14:26:00 +0000
Subject: Always allow inspect fallback

Always enable 'inspect fallback' in 'chrome://inspect/#devices'

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 chrome/browser/resources/inspect/inspect.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/resources/inspect/inspect.js b/chrome/browser/resources/inspect/inspect.js
--- a/chrome/browser/resources/inspect/inspect.js
+++ b/chrome/browser/resources/inspect/inspect.js
@@ -45,6 +45,7 @@ if ('browser-inspector' in queryParamsObject) {
 })();
 
 function isVersionNewerThanHost(version) {
+  if ((true)) return true;
   if (!HOST_CHROME_VERSION) {
     return false;
   }
@@ -566,7 +567,7 @@ function formatValue(data, property) {
 function addFavicon(row, data) {
   const favicon = document.createElement('img');
   if (data['faviconUrl']) {
-    favicon.src = data['faviconUrl'];
+    //favicon.src = data['faviconUrl'];
   }
   const propertiesBox = row.querySelector('.properties-box');
   propertiesBox.insertBefore(favicon, propertiesBox.firstChild);
-- 

